examples/compat/Coral.Element.js

```diff
@@ -63,7 +63,7 @@
         transform: Coral.transform.boolean,
         attributeTransform: Coral.transform.booleanAttr,
         sync: function() {
-          this.setAttribute('aria-disabled', this.disabled);
+          this[this.disabled ? 'setAttribute' : 'removeAttribute']('aria-disabled', this.disabled);
```

Comment:
What's the reason for changing this across all test files ?

Reply:
`[aria-disabled="false"]` isn't meaningful or necessary, so it's better to remove it.